### PR TITLE
SWAPI new test and validation data

### LINKED ISSUE #890: Update SWAPI test and validation data
- [ ] Added new test and validation data and update tests to reflect new files.
- [ ] Update HVENG to HVSCI
- [ ] Update HK Association

### PR DESCRIPTION
**Pull request by tech3371**
# Change Summary
closes #890 

## Overview
Added new SWAPI data and removed old test files.

## Updated Files
- imap_processing/cdf/config/imap_swapi_variable_attrs.yaml
   - format update
- imap_processing/swapi/l1/swapi_l1.py
   - update to use HVSCI now
   - update comment and HK association
- deleted old test data and added new data

## Testing
- imap_processing/tests/swapi/test_swapi_decom.py
- imap_processing/tests/swapi/test_swapi_l1.py
- imap_processing/tests/swapi/test_swapi_l2.py

### CODE REVIEW

File: imap_processing/tests/swapi/test_swapi_decom.py [hunk 1]
```diff
@@ -79,15 +74,26 @@ def test_swapi_hk_data(decom_test_data, swapi_l0_validation_data_path):
...
+        swapi_l0_validation_data_path / "idle_export_raw.SWP_HK_20240924_080204.csv",
         index_col="SHCOARSE",
     )
 
     grouped_data = group_by_apid(decom_test_data)
     hk_packets = grouped_data[SWAPIAPID.SWP_HK]
     first_data = hk_packets[0]
     validation_data = raw_validation_data.loc[first_data.data["SHCOARSE"].raw_value]
-
+    bad_keys = [
```
Comment (maxinelasp) [inline imap_processing/tests/swapi/test_swapi_decom.py:85]: Are these bad in the validation data? I see you have a TODO on ln 112, is that still accurate?

File: imap_processing/tests/swapi/test_swapi_l1.py [hunk 2]
```diff
@@ -49,19 +50,18 @@ def test_filter_good_data():
     bad_data_indices = filter_good_data(ds)
     assert len(bad_data_indices) == 36
 
-    # Check for bad MODE data, only HVENG is "good"
-    # TODO: update test when we update MODE from HVENG to HVSCI
+    # Check for bad MODE data, only HVSCI is "good"
     ds["mode"] = xr.DataArray(
         np.repeat(
-            [SWAPIMODE.LVENG.value, SWAPIMODE.LVSCI.value, SWAPIMODE.HVENG.value], 12
```
Comment (maxinelasp) [inline imap_processing/tests/swapi/test_swapi_l1.py:56]: Is HVENG no longer good? Why?

**Review by maxinelasp — APPROVED** [review]
LGTM overall, just had a few questions but I assume you know the answer and have already considered them

File: imap_processing/tests/swapi/test_swapi_l1.py (hunk 2 above)
Comment (tech3371) [reply imap_processing/tests/swapi/test_swapi_l1.py:56]: Yes. Before I didn't have HVSCI data which is actually what we want after launch. I left HVENG as placeholder until today.

File: imap_processing/tests/swapi/test_swapi_decom.py (hunk 1 above)
Comment (tech3371) [reply imap_processing/tests/swapi/test_swapi_decom.py:85]: Yes. I just updated them. I plan to ask for new data but it's low priority.

**Pull request merged**